### ISSUE TITLE
Magic/Physical Enhancer Rings bonus and combo

### ISSUE THREAD

**Issue by Darkelfen** [rathena/rathena#1445]
These items bonuses are wrong since they are giving a bonus which should be given when worn in combo.

re/item_db.txt

```
2963,Upgrade_Body_Ring,Upgrade Body Ring,4,10,,100,,,,1,0xFFFFFFFF,63,2,136,,0,0,0,{ bonus bAtkRate,5; bonus bAspdRate,5; },{},{}
2964,Amplified_Magical_Ring,Amplified Magical Ring,4,10,,100,,,,1,0xFFFFFFFF,63,2,136,,0,0,0,{ bonus bMatkRate,5; bonus bVariableCastrate,-5; },{},{}
```

it should be:

```
2963,Upgrade_Body_Ring,Upgrade Body Ring,4,10,,100,,,,1,0xFFFFFFFF,63,2,136,,0,0,0,{ bonus bAtkRate,5; },{},{}
2964,Amplified_Magical_Ring,Amplified Magical Ring,4,10,,100,,,,1,0xFFFFFFFF,63,2,136,,0,0,0,{ bonus bMatkRate,5; },{},{}
```

re/item_combo_db.txt

Regarding their combos they are non-existant.
Physical Enhancer Ring + Geffen Magic Robe: 5% Demi-human Resistance and 5% Delay Reduction.
Physical Enhancer Ring + Anti-Magic Suit: Aspd +5%.
Magical Enhancer Ring + Anti-Magic Suit: 5% Demi-human Resistance and 5% Delay Reduction.
Magical Enhancer Ring + Geffen Magic Robe: Reduces Variable Cast Rate by 5%.

it should be:

```
2963:15073,{ bonus bAspdRate,5; }
2963:15074,{ bonus bDelayRate,-5; bonus2 bSubRace,RC_DemiHuman,5; bonus2 bSubRace,RC_Player,5; }
2964:15073,{ bonus bDelayRate,-5; bonus2 bSubRace,RC_DemiHuman,5; bonus2 bSubRace,RC_Player,5; }
2964:15074,{ bonus bVariableCastrate,-5; }
```

Source: Divine-pride
[Physical Enhancer Ring](http://www.divine-pride.net/database/item/2963)
[Magical Enchancer Ring](http://www.divine-pride.net/database/item/2964)

**Comment by Lemongrass3110** [comment]
[Geffen Magic Robe](http://www.divine-pride.net/database/item/15074/geffen-magic-robe-1)
[Anti Magic Suit](http://www.divine-pride.net/database/item/15073/anti-magic-suit-1)

We need to check those, but your report looks good.

**Comment by Darkelfen** [comment]
Regarding Geffen Magic Robe is giving the bonus of cast reduction at +7 instead 
of at +9 as stated in DP's Korean descript. Anti Magic Suit script fits DP's description.

re/item_db.txt
```
15074,Geffen_Magic_Robe,Geffen Magic Robe,4,10,,1000,,40,,1,0xFFFFFFFF,63,2,16,,99,1,0,{ .@r = getrefine(); bonus bUseSPrate,-10-((.@r>=7) ? 5: 0); bonus bVariableCastrate,-15-((.@r>=7) ? 5: 0); bonus bNoCastCancel,1; },{},{}
```

according to DP description it should be:
```
15074,Geffen_Magic_Robe,Geffen Magic Robe,4,10,,1000,,40,,1,0xFFFFFFFF,63,2,16,,99,1,0,{ .@r = getrefine(); bonus bUseSPrate,-10-((.@r>=7) ? 5: 0); bonus bVariableCastrate,-15-((.@r>=9) ? 5: 0); bonus bNoCastCancel,1; },{},{}
```

**Comment by aleos89** [comment]
Fixed in 07eb5ea.
